java/src/main/java/com/saucelabs/simplesauce/SauceSession.java

```diff
@@ -65,23 +64,25 @@ public SauceSession(SauceOptions options, RemoteDriverInterface remoteManager, E
         this.environmentManager = environmentManager;
     }
 
+    public WebDriver getDriver() {
```

Comment:
@joshmgrant I think that if you use Lombok, we don't need this. 
Keep this `@Getter public WebDriver webDriver;` but make it `public` and then with Lombok, the user should be able to just do `sauce.getwebDriver()` and it will automatically work with magic.

Reply:
this can be changed I think